pkg/oauth/util/discovery.go

```diff
@@ -51,14 +50,13 @@ type OauthAuthorizationServerMetadata struct {
 // validate configuration using LoadOAuthMetadataFile
 
 func getOauthMetadata(masterPublicURL string) OauthAuthorizationServerMetadata {
-	config := osinserver.NewDefaultServerConfig()
 	return OauthAuthorizationServerMetadata{
 		Issuer:                masterPublicURL,
 		AuthorizationEndpoint: urls.OpenShiftOAuthAuthorizeURL(masterPublicURL),
 		TokenEndpoint:         urls.OpenShiftOAuthTokenURL(masterPublicURL),
 		// Note: this list is incomplete, which is allowed per the draft spec
 		ScopesSupported:               scope.DefaultSupportedScopes(),
-		ResponseTypesSupported:        config.AllowedAuthorizeTypes,
+		ResponseTypesSupported:        osin.AllowedAuthorizeType{osin.CODE, osin.TOKEN},
 		GrantTypesSupported:           osin.AllowedAccessType{osin.AUTHORIZATION_CODE, osin.AccessRequestType("implicit")}, // TODO use config.AllowedAccessTypes once our implementation handles other grant types
```

Comment:
Should this comment get deleted as well? 🙂

Reply:
Probably.